docs/contributing.md

```diff
@@ -105,3 +105,21 @@ Then:
 ```
 Click the link it outputs. As you save changes to files modified in your editor,
 the browser will automatically show the new content.
+
+
+How to File an Issue
+--------------------
+
+[New installer issue](https://pulp.plan.io/projects/pulp/issues/new)
```

Comment:
I think we need to call out the setting of the 'Installer' as the Category before we even link the user to it.

Reply:
I mean something like this:

```
To file a new issue set the Category to 'Installer' when filing [here](https://pulp.plan.io/projects/pulp/issues/new).
```